vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -146,6 +146,8 @@
     </Link>
     <Midl>
       <AdditionalIncludeDirectories>$(ReactNativeWindowsDir)Microsoft.ReactNative\Views\cppwinrt;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <!-- #8430: This is needed to allow modules targeting older SDKs to consume the geneerated WinMD -->
+      <AdditionalOptions>/noattributename</AdditionalOptions>
```

Comment:
Thanks for taking this on. I thought we were going to go with the other approach of not even producing the doc string info outside of the context of CI / producing the winmd api docs. I think that's cleaner than using an undocumented MIDL flag, mind changing it?

Reply:
This issue from what I understand seems to be related to general attributes rather the doc string attribute specifically.  I worry that if we special case to doc strings, we may see the same issue in other contexts.

Reply:
the only attributes we have are the ones we created for doc purposes. Are you worried about us creating more custom attributes in the future? It's possible we would at some point but I'm not super worried about it right now

Reply:
Yes, it is something where an attribute could be added to us, or inside of a community module, and we can see a break with no signal (unless we keep old versions in our repo).